google/cloud/security/common/data_access/sql_queries/create_tables.py

```diff
@@ -115,3 +115,30 @@
 """
 
 # TODO: Add a RAW_GROUP_MEMBERS_TABLE.
+
+CREATE_BUCKETS_TABLE = """
+    CREATE TABLE `{0}` (
+        `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+        `project_number` bigint(20) NOT NULL,
+        `bucket_id` varchar(255) DEFAULT NULL,
+        `bucket_name` varchar(255) DEFAULT NULL,
+        `bucket_kind` varchar(255) DEFAULT NULL,
+        `bucket_storage_class` varchar(255) DEFAULT NULL,
+        `bucket_location` varchar(255) DEFAULT NULL,
+        `bucket_create_time` datetime DEFAULT NULL,
+        `bucket_update_time` datetime DEFAULT NULL,
+        `bucket_selflink` varchar(255) DEFAULT NULL,
+        `bucket_lifecycle_raw` json DEFAULT NULL,
+        `raw_bucket` json DEFAULT NULL,
+        PRIMARY KEY (`id`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+"""
+
+CREATE_RAW_BUCKETS_TABLE = """
```

Comment:
The idea behind this table was to have a map of project_number: all_project_buckets_json, in case there was a need for this data in the future or diffing. I'm pretty unsure if this will ever be used, so I can just remove it and adding it in the future if we need it is not a biggie.